.github/workflows/packer.yml

```diff
@@ -0,0 +1,32 @@
+name: Packer Validate
+
+on:
+  pull_request:
+    paths:
+      - packer/**
+
+env:
+  PACKER_VERSION: "1.8.7"
+
+jobs:
+  packer-validate:
+    runs-on: ubuntu-latest
+    name: Run Packer
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Setup packer
+        uses: hashicorp/setup-packer@main
+        id: setup
+        with:
+          version: ${{ env.PACKER_VERSION }}
+
+      - name: Validate Packer Templates
+        id: validate
+        working-directory: packer
+        run: |
+            for template in `ls | grep ".json"`; do
+                echo $template
```

Comment:
Do we need to echo the entire template or was this for debugging purpose?

Reply:
It is just echoing the name of the template, not the content.

Reply:
```
jenkins-agent-al2-arm64.json
The configuration is valid.
jenkins-agent-al2-x64.json
The configuration is valid.
jenkins-agent-al2023-arm64.json
The configuration is valid.
jenkins-agent-al2023-x64.json
The configuration is valid.
jenkins-agent-macos12-x64.json
The configuration is valid.
jenkins-agent-ubuntu2004-x64.json
The configuration is valid.
jenkins-agent-win2019-x64-gradle-check.json
The configuration is valid.
jenkins-agent-win2019-x64.json
The configuration is valid.

```